File: verify-check/AOJ-NTL-1-D.test.cpp
#define PROBLEM "https://onlinejudge.u-aizu.ac.jp/problems/NTL_1_D"
#include <iostream>
#include "num_theory/Eulers_totient_function.cpp"
using namespace std;
int main() {
    int N;
    cin >> N;
    cout << EulersTotientFunction(N) << endl;
    return 0;
}

File: verify-check/AOJ-NTL-1-D.test.py
# verification-helper: PROBLEM https://onlinejudge.u-aizu.ac.jp/courses/library/6/NTL/1/NTL_1_D

from num_theory.Eulers_totient_function import Eulers_totient_function

N = int(input())
print(Eulers_totient_function(N))
